gnomad_qc/v3/annotations/generate_qc_annotations.py

```diff
@@ -304,7 +311,7 @@ def export_transmitted_singletons_vcf():
 
 def run_vep(vep_version: str = "101") -> hl.Table:
     """
-    Returns a table with a VEP annotation for each variant in the raw MatrixTable.
+    Return a table with a VEP annotation for each variant in the raw MatrixTable.
```

Comment:
Is this autopep? Or did you make these changes? If autopep, thats awesome

Reply:
No unfortunately this one is me

Reply:
haha 🤦  -- on Fridays, I'm an optimist -- that would've been really impressive